### PR TITLE
[cleanup] fix compiler warnings

### PR DESCRIPTION
**Pull request by mkortstiege**
Silents some compiler warnings.

### CODE REVIEW

File: xbmc/utils/SortUtils.cpp [hunk 1]
```diff
@@ -245,7 +245,7 @@ std::string ByRating(SortAttribute attributes, const SortItem &values)
 
 std::string ByUserRating(SortAttribute attributes, const SortItem &values)
 {
-  return StringUtils::Format("%d %s", values.at(FieldUserRating).asInteger(), ByLabel(attributes, values).c_str());
+  return StringUtils::Format("%d %s", (int)values.at(FieldUserRating).asInteger(), ByLabel(attributes, values).c_str());
```
Comment (Montellese) [inline xbmc/utils/SortUtils.cpp]: `static_cast<int>()` please

**Comment by mkortstiege** [comment]
Updated to use `static_cast<int>()` as suggested.

**Comment by mkortstiege** [comment]
jenkins build this please.

**Comment by mkortstiege** [comment]
Any objections to shove this in within the October merge window?

**Comment by mkortstiege** [comment]
jenkins build and merge

**Pull request merged**